CHANGELOG.md

```diff
@@ -4,6 +4,35 @@
 
 💥 Breaking changes:
 
+- Make radios and checkboxes components easier to link to from the error summary component
+
+  The first input in the radios and checkboxes will have an `id` and `name` attribute that match, allowing for easier generation of error summary links.
+
+  You should check that clicking an error link for a radios or checkboxes results in the first input being focused.
+
+  To migrate you can update your error summary so it links to the first element, for example if you are using an `idPrefix` option of `question`, your error summary can now link to `#question` instead of `#question-1`.
```

Comment:
Users might also be overriding the `id` of the first radio/checkbox, which I think is what we suggested in #929. In that case, they could remove the `id` as it won't be doing anything.

Reply:
I figured since this wasnt required that I'd leave the up to people themselves rather than potentially confusing things... maybe should just have this in

Reply:
I tried this and I'm worried that there's just a lot of steps being described that could confuse people so I wanted to focus on the things they definitely need to do...

Reply:
nevermind I've included it after all..